.github/workflows/push_tests.yml

```diff
@@ -37,8 +37,8 @@ jobs:
           python-version: "3.8"
       - name: Install dependencies
         run: |
-          pip install -e .
-          pip install huggingface_hub
+          python -m venv /opt/venv && export PATH="/opt/venv/bin:$PATH"
+          python -m uv pip install -e [quality,test]
```

Comment:
Why do we have to rely on a full-fledged installation to fetch pipeline usage numbers?

Reply:
We need transformers and torch installed so that we can get the correct pipeline modules and not dummies. I just reverted it back to what it was before.